client/src/controllers/SwapController.ts

```diff
@@ -215,6 +217,20 @@ export class SwapController extends Controller<
     this.replace(url + queryString);
   }
 
+  reflectParams(url: string) {
```

Comment:
```suggestion
  /**
   * Allow for swapped URLs to add a history record so that the back button
   * can be used to navigate to the previous state set by this controller.
   */
  reflectParams(url: string) {
```

I know the name helps but a suggestion for a bit more context when others are navigating this code.